Symfony/CS/Fixer/Symfony/PhpdocInlineTagFixer.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * Fix inline tags and make inheritdoc tag always inline.
+ */
+final class PhpdocInlineTagFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isTokenKindFound(T_DOC_COMMENT);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        foreach ($tokens as $token) {
+            if (!$token->isGivenKind(T_DOC_COMMENT)) {
+                continue;
+            }
+
+            $content = $token->getContent();
+
+            // Move `@` inside tag, for example @{tag} -> {@tag}, replace multiple curly brackets,
+            // remove spaces between '{' and '@', remove 's' at the end of tag.
+            // Make sure the tags are written in lower case, remove white space between end
+            // of text and closing bracket and between the tag and inline comment.
+            $content = preg_replace_callback(
+                '#(@{+|{+[ \t]*@)[ \t]*(example|id|internal|inheritdoc|link|source|toc|tutorial)s*([^}]*)(}*)#iu',
```

Comment:
I thought `u` is for UTF8 (PCRE_UTF8) right? Since PSR requires all code to be UTF8 encoded this might be a useful flag. Maybe I can puzzle a test together, but is there a need to not set this flag?